pkg/ktexporter/ktmetrics/ktmetrics.go

```diff
@@ -0,0 +1,508 @@
+package ktmetrics
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
```

Comment:
Use os.ReadDir instead of ioutil.ReadDir as ioutil is deprecated.

建议的更改:

- 	"io/ioutil"
+ 	"io/fs"
- 	containerFiles, err := ioutil.ReadDir(containerdDir)
+ 	containerFiles, err := os.ReadDir(containerdDir)
- 	files, err := ioutil.ReadDir(cgroupDir)
+ 	files, err := os.ReadDir(cgroupDir)